src/plinder/eval/docking/utils.py

```diff
@@ -85,10 +85,17 @@ def from_files(
         else:
             entity = io.LoadPDB(receptor_file.as_posix(), fault_tolerant=True)
 
-        ligand_views = [
-            io.LoadEntity(str(ligand_sdf_file), format="sdf").Select("ele != H")
-            for ligand_sdf_file in ligand_files
-        ]
+        ligand_views = []
+        for i, ligand_file in enumerate(ligand_files):
+            ligand_entity = io.LoadEntity(str(ligand_file), format="sdf")
+
+            # rename ligand chain to have different chain names for each ligand
+            # this is necessary for ost to not complain about duplicate chain names
+            editor = ligand_entity.EditXCS()
+            editor.RenameChain(list(ligand_entity.chains)[0], f"LIG_{i}")
```

Comment:
not sure about this chain label convention - as it is normally a single letter - I suppose a lowercase letter might be sufficient or just a number might also work. But as this is only for scoring - also happy with whatever works!
WDYT @Ninjani ?

Reply:
yeah that was just me using the first thing that came to my mind 😅 happy to use a better convention!

Reply:
I think it's fine as we don't save the molecules anywhere!